src/requirements.txt

```diff
@@ -1,3 +1,6 @@
 cumulus-message-adapter==2.0.3
 cumulus-message-adapter-python==2.2.0
+jsonpath-ng==1.4.2
+jsonschema==2.6.0
 mandible @ git+https://github.com/asfadmin/mandible@v0.4.0
+ply==3.11
```

Comment:
I don not see ply being used.. If its a dependency of anther package we shouldnt pin it imo .

Reply:
👍

Reply:
Like I think it shouldn't be in the requirements?